pandas/tests/groupby/test_function.py

```diff
@@ -490,6 +490,25 @@ def test_ops_general():
             raise
 
 
+@pytest.mark.parametrize('as_index', [True, False])
+def test_groupby_std(as_index):
```

Comment:
Hmm is there any reason why we need a dedicated test for std here? We could alternately:

  1. Add std to  test_groupby_as_index_agg in test_groupby OR
  2. Parametrize as_index test_regression_whitelist_methods in test_whitelist

The latter may be more comprehensive

Reply:
One of these probably makes sense. I am starting down the path of (2) and just want to note that `as_index=False` is only valid for `axis=0` (`axis` is parameterized there as well ... should I write it to parameterize on `as_index` but just `pass` cases where `axis=1` and `as_index=False`?).

Reply:
Yes you can pytest.skip those

Reply:
@WillAyd Having some difficulty figuring out how to correctly construct the expected dataframes, the index column is giving me trouble, but will take a look again later this week.

Reply:
Open to tips on this... if you add `'as_index': as_index` to the groupby_kwargs, tests start to fail... I'm sure I can figure it out but probably need to take a deeper dive later.